### PR TITLE
Remove unnecessary awaits

### PR DESCRIPTION
**Pull request by annagrin**
Remove unnecessary awaits and  async keywords.

### CODE REVIEW

**Comment by annagrin** [comment]
@natebosch I hope I got the idea right on where asyncs/awaits can be safely removed. Appreciate a review!

File: dwds/lib/src/debugging/classes.dart [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.import 'dart:async';
+// BSD-style license that can be found in the LICENSE file.
```
Comment (annagrin) [inline dwds/lib/src/debugging/classes.dart:3]: looks like accidental change propagated by copy and paste in places:)

File: dwds/debug_extension_mv3/tool/update_dev_files.dart [hunk 2]
```diff
@@ -2,7 +2,6 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'dart:async';
```
Comment (annagrin) [inline dwds/debug_extension_mv3/tool/update_dev_files.dart:5]: I wonder why the analyzer does not catch those...

File: dwds/lib/src/debugging/classes.dart [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.import 'dart:async';
```
Comment (annagrin) [inline dwds/lib/src/debugging/classes.dart:3]: looks like accidental change propagated by copy and paste in places:)

File: dwds/debug_extension_mv3/tool/update_dev_files.dart (hunk 2 above)
Comment (natebosch) [reply dwds/debug_extension_mv3/tool/update_dev_files.dart:5]: Hmm, it should

https://dart.dev/tools/diagnostic-messages#unnecessary_import

@srawlins - do you know why these aren't flagged?

**Review by natebosch — APPROVED** [review]

File: dwds/debug_extension_mv3/tool/update_dev_files.dart (hunk 2 above)
Comment (srawlins) [reply dwds/debug_extension_mv3/tool/update_dev_files.dart:5]: Not offhand. I've seen `unnecessary_import` call out other `dart:async` imports.

**Pull request merged**